example/plugins/c-api/secure_link/secure_link.cc

```diff
@@ -128,8 +129,8 @@ TSRemapDoRemap(void *ih, TSHttpTxn rh, TSRemapRequestInfo *rri)
     snprintf(&hash[i * 2], sizeof(hash) - (i * 2), "%02x", md[i]);
   }
   time(&t);
-  e = (NULL == expire ? 0 : strtol(expire, NULL, 16));
-  i = TSREMAP_DID_REMAP;
+  e      = (NULL == expire ? 0 : strtol(expire, NULL, 16));
```

Comment:
Missed `nullptr`?

Reply:
OK I addressed all of these I think in a new (big) commit.